sys/dev/drm/panfrost/panfrost_gem.c

```diff
@@ -96,6 +96,9 @@ panfrost_gem_free_object(struct drm_gem_object *obj)
 	if (bo->pages) {
 		for (i = 0; i < bo->npages; i++) {
 			m = bo->pages[i];
+			if (m == NULL)
+				continue;
```

Comment:
why not break ?

Reply:
Because the tiny increase in performance for an error case doesn't matter and it means we don't depend on the array being initialised in order

Reply:
what if some one request millions of pages? panfrost allocates in order only, similar to cma
https://github.com/CTSRD-CHERI/cheribsd/blob/dev/sys/dev/drm/freebsd/drm_gem_cma_helper.c#L71

Reply:
Then we shouldn't have allocated an array of a million page pointers for bo->pages in the first place